docker/1.3.0/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,93 @@
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
+# NCCL_VERSION=2.4.7, CUDNN_VERSION=7.6.2.24
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y  --allow-downgrades --allow-change-held-packages --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        cmake \
+        libgomp1 \
+        libibverbs-dev \
+        curl \
+        git \
+        wget \
+        vim \
+        jq \
+        libsm6 \
+        libxext6 \
+        libxrender-dev \
+        build-essential \
+        zlib1g-dev \
+        libglib2.0-0 \
+        libgl1-mesa-glx \
+        openjdk-8-jdk-headless
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.6.6
+
+# Install OpenSSH. Allow OpenSSH to talk to containers without asking for confirmation
+RUN apt-get install -y --no-install-recommends openssh-client openssh-server && \
+    mkdir -p /var/run/sshd && \
+    cat /etc/ssh/ssh_config | grep -v StrictHostKeyChecking > /etc/ssh/ssh_config.new && \
+    echo "    StrictHostKeyChecking no" >> /etc/ssh/ssh_config.new && \
+    mv /etc/ssh/ssh_config.new /etc/ssh/ssh_configs
+
+RUN curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+    chmod +x ~/miniconda.sh && \
+    ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda update conda && \
+    /opt/conda/bin/conda install -y python=$PYTHON_VERSION \
+	                              numpy==1.16.4 \
+	                              scipy==1.3.0 \
+	                              ipython==7.7.0 \
+	                              mkl==2019.4 \
+	                              mkl-include==2019.4 \
+	                              cython==0.29.12 \
+	                              typing==3.6.4 && \
+	 /opt/conda/bin/conda clean -ya
+ENV PATH /opt/conda/bin:$PATH
+
+ARG PYTORCH_VERSION=1.3.0
+ARG TORCHVISION_VERSION=0.4.1
+ARG MMS_VERSION=1.0.7
```

Comment:
Please update this to the latest 1.0.8 everywhere.

Reply:
done